daemon/mgr/container.go

```diff
@@ -1488,6 +1496,90 @@ func (mgr *ContainerManager) parseBinds(ctx context.Context, meta *ContainerMeta
 	return nil
 }
 
+func (mgr *ContainerManager) setMountPointDiskQuota(ctx context.Context, c *ContainerMeta) error {
+	var (
+		qid        uint32
+		setQuotaID bool
+	)
+
+	if c.Config.QuotaID != "" {
+		id, err := strconv.Atoi(c.Config.QuotaID)
```

Comment:
In the future, we will move this kind of validation out of package mgr. And it should be in the scope of bridge layer.